dashboard/app/controllers/resources_controller.rb

```diff
@@ -1,6 +1,32 @@
 class ResourcesController < ApplicationController
+  before_action :require_levelbuilder_mode, except: [:index, :show]
```

Comment:
this is fine as it is for now, but reminder when we go to add UI tests that this will need to change to `require_levelbuilder_mode_or_test_env` in order to pass UI tests on drone.

Reply:
Right. I'll update that now as I'm already editing this file